pkg/apis/servicecatalog/types.go

```diff
@@ -797,3 +796,25 @@ type SecretKeyReference struct {
 	// The key of the secret to select from.  Must be a valid secret key.
 	Key string
 }
+
+// ObjectReference contains enough information to let you inspect or modify the referred object.
```

Comment:
referenced object?

Reply:
yeah, this text was actually copied from `k8s.io/api/core/v1`, but I will fix it.